memcached.c

```diff
@@ -8919,11 +8943,15 @@ int main (int argc, char **argv) {
     setbuf(stderr, NULL);
 
     char *shortopts =
+#ifndef DISABLE_UNIX_SOCKET
```

Comment:
... but leave the option parsers in there. remove all the ifdefs from getopt stuff.